pkg/controller/argocd/util.go

```diff
@@ -1204,3 +1207,20 @@ func initK8sClient() (*kubernetes.Clientset, error) {
 
 	return k8sClient, nil
 }
+
+// getLogLevel returns the log level for a specified component if it is set or returns the default log level if it is not set
+func getLogLevel(logField string) string {
+
+	if logField == "" {
+		return common.ArgoCDDefaultLogLevel
+	} else {
+		switch logField {
+		case
+			"info",
+			"warn",
+			"error":
```

Comment:
Is `debug` a supported loglevel?

Reply:
According to https://argoproj.github.io/argo-cd/operator-manual/server-commands/argocd-server/ (and applies to each of the components), it looks like the only options are `info`,`warn`, and `error`

Reply:
I see this from the reference you provided.   Please double check
```
      --loglevel string                               Set the logging level. One of: debug|info|warn|error (default "info")
```

Reply:
Ah. Must have overlooked this. This is fixed now.